assets/js/hooks/session.js

```diff
@@ -81,6 +82,7 @@ const Session = {
     this.viewOptions = null;
     this.keyBuffer = new KeyBuffer();
     this.lastLocationReportByClientId = {};
+    this.history = new History();
```

Comment:
Nitpick: history is a very generic term and could be easily confused for changes history, so I would use a more specific name everywhere, such as `CursorHistory` :)